vscode-lean4/src/leanclient.ts

```diff
@@ -227,7 +230,8 @@ export class LeanClient implements Disposable {
         if (doc.languageId === 'lean') {
             // Only change the id for *visible* documents,
             // because this closes and then reopens the document.
-            await languages.setTextDocumentLanguage(doc, 'lean4')
+            await languages.setTextDocumentLanguage(doc, 'lean4');
+            this.didSetLanguageEmitter.fire(undefined);
```

Comment:
Oh I just noticed this, since it's `EventEmitter<string>` maybe `fire` with `'lean4'` here? Or change the type to `EventEmitter<undefined>`.

Reply:
Oh right, 'lean4' makes perfect sense here. Fixed.